x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages.tsx

```diff
@@ -7,32 +7,32 @@
 
 import React, { Fragment } from 'react';
 
-import { useValues } from 'kea';
+import { useValues, useActions } from 'kea';
 
-import { EuiCallOut, EuiCallOutProps, EuiSpacer } from '@elastic/eui';
+import { EuiCallOut, EuiCallOutProps, EuiSpacer, EuiGlobalToastList } from '@elastic/eui';
 
+import { FLASH_MESSAGE_TYPES, DEFAULT_TOAST_TIMEOUT } from './constants';
 import { FlashMessagesLogic } from './flash_messages_logic';
 
-const FLASH_MESSAGE_TYPES = {
-  success: { color: 'success' as EuiCallOutProps['color'], icon: 'check' },
-  info: { color: 'primary' as EuiCallOutProps['color'], icon: 'iInCircle' },
-  warning: { color: 'warning' as EuiCallOutProps['color'], icon: 'alert' },
-  error: { color: 'danger' as EuiCallOutProps['color'], icon: 'alert' },
+export const FlashMessages: React.FC = ({ children }) => {
+  return (
+    <>
+      <Callouts>{children}</Callouts>
+      <Toasts />
+    </>
+  );
 };
 
-export const FlashMessages: React.FC = ({ children }) => {
+export const Callouts: React.FC = ({ children }) => {
   const { messages } = useValues(FlashMessagesLogic);
 
-  // If we have no messages to display, do not render the element at all
-  if (!messages.length) return null;
-
   return (
-    <div data-test-subj="FlashMessages">
+    <div aria-live="polite" role="region" data-test-subj="FlashMessages">
```

Comment:
I'm super excited for this enhancement! I noticed while inspecting `EuiGlobalToastList` that they were using these aria-live/role props and announcing new messages, and was like "ohh, we should be doing that as well for our callouts!"

This works fairly well from my testing. I will note that the removed conditional rendering is required - for `aria-live` regions to work correctly, the div container must be present initially for screen readers to detect that its content has changed.